src/group.rs

```diff
@@ -141,143 +149,287 @@ impl GroupCreateOpts {
+    /// let remove_result = sdk.group_remove_members(&group_id, &vec![user]).await?;
+    /// let removed_members: Vec<UserId> = remove_result.succeeded().to_vec();
+    /// let failures: Vec<GroupAccessEditErr> = remove_result.failed().to_vec();
+    /// # Ok(())
+    /// # }
+    /// ```
     ///
-    /// # Returns
-    /// `Result<GroupAccessEditResult>` List of users that were removed. Also contains the users that failed to be removed
-    ///    and the reason they were not.
+    /// # Errors
+    /// This operation supports partial success. If the request succeeds, then the resulting `GroupAccessEditResult`
+    /// will indicate which removals succeeded and which failed, and it will provide an explanation for each failure.
     async fn group_remove_members(
         &self,
         id: &GroupId,
         revoke_list: &[UserId],
     ) -> Result<GroupAccessEditResult>;
 
-    /// Add the users as admins of a group.
+    /// Adds administrators to a group.
+    ///
+    /// Returns lists of successful and failed additions.
     ///
     /// # Arguments
-    /// - `id` - ID of the group to add admins to
-    /// - `users` - The list of users that will be added to the group as admins.
-    /// # Returns
-    /// GroupAccessEditResult, which contains all the users that were added. It also contains the users that were not added and
-    ///   the reason they were not.
+    /// - `id` - ID of the group to add administrators to
+    /// - `users` - List of users to add as group administrators
+    ///
+    /// # Examples
+    /// ```
+    /// # async fn run() -> Result<(), ironoxide::IronOxideErr> {
+    /// # use ironoxide::prelude::*;
+    /// # let sdk: IronOxide = unimplemented!();
+    /// # use std::convert::TryFrom;
+    /// let group_id = GroupId::try_from("empl412")?;
+    /// let user = UserId::try_from("colt")?;
+    /// let add_result = sdk.group_add_admins(&group_id, &vec![user]).await?;
+    /// let new_admins: Vec<UserId> = add_result.succeeded().to_vec();
+    /// let failures: Vec<GroupAccessEditErr> = add_result.failed().to_vec();
+    /// # Ok(())
+    /// # }
+    /// ```
+    ///
+    /// # Errors
+    /// This operation supports partial success. If the request succeeds, then the resulting `GroupAccessEditResult`
+    /// will indicate which additions succeeded and which failed, and it will provide an explanation for each failure.
     async fn group_add_admins(
         &self,
         id: &GroupId,
         users: &[UserId],
     ) -> Result<GroupAccessEditResult>;
 
-    /// Remove a list of users as admins from the group.
+    /// Removes administrators from a group.
+    ///
+    /// Returns lists of successful and failed removals.
     ///
     /// # Arguments
-    /// - `id` - ID of the group
-    /// - `revoke_list` - List of user IDs to remove as admins
+    /// - `id` - ID of the group to remove administrators from
+    /// - `revoke_list` - List of users to remove as group administrators
```

Comment:
I would almost call this `to_remove` or something. Perhaps this outside the scope of the doc rework though.